nodejs/src/examples/futures/get-wallet-information.js

```diff
@@ -0,0 +1,16 @@
+const axios = require('axios');
+const { getFuturesUrl, getAuthHeaders } = require('../../utils/common');
+
+const getWalletInformation = async () => {
+  const endpoint = '/api/v2.1/user/wallet';
```

Comment:
@tinganliu since we basically want to create examples for the latest version of apis, would it be possible to define constants for api versions (spot & futures) so we can change the api version for all cases in one place?